### PR TITLE
cni: add servicePort CRD

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

File: pkg/apis/service/install/install.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Antrea Authors
```
Comment (zwtop) [inline pkg/apis/service/install/install.go:1]: Copyright  Antrea Authors?

File: pkg/apis/service/v1alpha1/types.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=sp
+// +kubebuilder:subresource:status
```
Comment (zwtop) [inline pkg/apis/service/v1alpha1/types.go:29]: CRD doesn't have a field named status

File: pkg/apis/service/install/install.go (hunk 1 above)
Comment (qingwusunny) [reply pkg/apis/service/install/install.go]: already modify

File: pkg/apis/service/v1alpha1/types.go (hunk 2 above)
Comment (qingwusunny) [reply pkg/apis/service/v1alpha1/types.go]: done

**Review by wu-cl — DISMISSED** [review]

File: pkg/apis/service/doc.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +groupName=internal.everoute.io
```
Comment (zwtop) [inline pkg/apis/service/doc.go:18]: nit: group name should be `service.everoute.io`? or just remove the comment.

**Review by zwtop — DISMISSED** [review]

File: pkg/apis/service/doc.go (hunk 3 above)
Comment (qingwusunny) [reply pkg/apis/service/doc.go]: done

**Review by zwtop — APPROVED** [review]

**Review by wu-cl — APPROVED** [review]

**Pull request merged**